librz/core/cbin.c

```diff
@@ -931,10 +938,46 @@ static char *construct_reloc_name(RZ_NONNULL RzBinReloc *reloc, RZ_NULLABLE cons
 	return rz_strbuf_drain(buf);
 }
 
-static void set_bin_relocs(RzCore *r, RzBinReloc *reloc, ut64 addr, Sdb **db, char **sdb_module) {
+static void reloc_set_flag(RzCore *r, RzBinReloc *reloc, const char *prefix, ut64 flag_addr) {
 	int bin_demangle = rz_config_get_i(r->config, "bin.demangle");
 	bool keep_lib = rz_config_get_i(r->config, "bin.demangle.libs");
 	const char *lang = rz_config_get(r->config, "bin.lang");
+	char *reloc_name = construct_reloc_name(reloc, NULL);
+	if (!reloc_name || !*reloc_name) {
```

Comment:
Use RZ_STR_ISEMPTY or IS_NULLSTR

Reply:
I think IS_NULLSTR is deprecated, better to use RZ_STR_ISEMPTY